apps/website/src/pages/about/tech/commands.tsx

```diff
@@ -61,6 +61,10 @@ const AboutTechPage: NextPage = () => {
     ),
   });
 
+  const [selectedCamera, setSelectedCamera] = useState<keyof typeof presets>(
+    Object.keys(presets)[0] as keyof typeof presets,
```

Comment:
As to avoid using an `as` type-cast here, the `typeSafeObjectKeys` util should help